src/color.rs

```diff
@@ -82,44 +82,34 @@ fn normalize_hue(hue: f32) -> f32 {
 
 /// A color with red, green, blue, and alpha components, in a byte each.
 #[derive(Clone, Copy, PartialEq, Debug)]
-pub struct RGBA {
+pub struct Rgba {
```

Comment:
Maybe we should call this `LegacyRgba` or somesuch?

Reply:
:+1: Makes more sense, fixing.